package.json

```diff
@@ -41,13 +41,13 @@
   "dependencies": {
     "@applitools/eyes-sdk-core": "11.0.2",
     "@applitools/eyes-webdriverio": "5.12.0",
-    "@wdio/allure-reporter": "^7.7.3",
-    "@wdio/cli": "^7.7.4",
-    "@wdio/devtools-service": "^7.7.4",
-    "@wdio/local-runner": "^7.7.4",
-    "@wdio/mocha-framework": "7.7.4",
-    "@wdio/selenium-standalone-service": "7.7.4",
-    "@wdio/sync": "7.7.4",
+    "@wdio/allure-reporter": "^7.14.1",
+    "@wdio/cli": "^7.14.1",
+    "@wdio/devtools-service": "^7.14.1",
+    "@wdio/local-runner": "^7.14.1",
+    "@wdio/mocha-framework": "7.14.1",
+    "@wdio/selenium-standalone-service": "7.14.1",
+    "@wdio/sync": "7.14.1",
```

Comment:
title says **update wdio packages to 7.16.6** but you uses 7.14.1

    "webdriverio": "7.7.4", should be updated as well

Reply:
done